integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/steps/Hooks.java

```diff
@@ -73,71 +60,75 @@ public void before(Scenario scenario) {
-                        final String bridgeId = bridgeContext.getId();
-                        BridgeResponse bridge = BridgeResource.getBridgeDetails(token, bridgeId);
-                        if (bridge.getStatus() == ManagedResourceStatus.READY) {
-                            ProcessorListResponse processorList = ProcessorResource.getProcessorList(
-                                    token,
-                                    bridgeId);
-                            if (processorList.getSize() > 0) {
-                                processorList.getItems().parallelStream().forEach(
-                                        p -> {
-                                            String processorId = p.getId();
-                                            ProcessorResource.deleteProcessor(token, bridgeId, processorId);
-                                            Awaitility.await()
-                                                    .atMost(Duration.ofMinutes(4))
-                                                    .pollInterval(Duration.ofSeconds(5))
-                                                    .untilAsserted(
-                                                            () -> assertThat(ProcessorResource.getProcessorList(token, bridgeId).getItems())
-                                                                    .as("waiting until Processor `%s` of the Bridge `%s` is deleted", processorId, bridgeId)
-                                                                    .noneMatch(processor -> Objects.equals(processor.getId(), processorId)));
-                                        });
-                            }
-                        }
-                        switch (bridge.getStatus()) {
-                            case ACCEPTED:
-                            case PROVISIONING:
-                            case READY:
-                            case FAILED:
-                                try {
-                                    BridgeResource.deleteBridge(token, bridgeId);
-                                    Awaitility.await()
-                                            .atMost(Duration.ofMinutes(4))
-                                            .pollInterval(Duration.ofSeconds(5))
-                                            .untilAsserted(
-                                                    () -> assertThat(BridgeResource.getBridgeList(token).getItems()).as("waiting until Bridge `%s` is deleted", bridgeId)
-                                                            .noneMatch(b -> Objects.equals(b.getId(), bridgeId)));
-                                } catch (Exception e) {
-                                    throw new RuntimeException("Unable to delete bridge with id " + bridgeId, e);
-                                }
-                            default:
-                                break;
-                        }
-                    });
-        }
-    }
+    /*
```

Comment:
Does this mean that the integration tests will leave a Bridge instance(s) behind?

Reply:
@winklerm Yes, it will leave instances behind.
The cleanup procedure fails as the manager is not properly connected with operator (new APIs are still in development) and thus the scenario fails too.

Reply:
Ok, thanks! I was just wondering if merging the code in this state could destabilize the environment, e.g. the dev cluster would be creating more and more Bridge instances that would never be deleted (and possibly some tests might fail on it)?

Bus since the PR is marked as do not merge, I guess it is fine now and we would just wait with merging until the clean up works. Wdyt?